README.md

```diff
@@ -12,38 +12,92 @@ The `amazon_cloud_code_generator` relies on the CloudFormation client. Hence, th
+    │       │       │   ├── aliases
+    │       │       │   └── tasks
+    │       │       │       └── main.yml
+    │       │       ├── ...
+    │       └── unit
+    │           ├── constraints.txt
+    │           ├── module_utils
+    │           │   ├── test_core.py
+    │           │   └── test_utils.py
+    │           └── requirements.txt
+    ```
+
+- `templates/` - Hosts the templates files that will be used to generate each collection's plugin. 
+    ```
+    ├── default_module.j2
+    └── header.j2
+    ```
+
+The modules generated by the `amazon_cloud_code_generator/` will be made availabale in the `cloud` subdirectory by default. You can use the following command to do so:
+
+```python -m amazon_cloud_code_generator.cmd.refresh_modules```
+
+A different target directory can be specified by using:
+
+```python -m amazon_cloud_code_generator.cmd.refresh_modules --target-dir /somewhere/else```
 
 When available, the documented EXAMPLE block of the generated modules can be updated by using the following command:
 
-```tox -e refresh_examples --target-dir /somewhere/else```
-
-This information is scraped from the tests/ directory.
+```python -m amazon_cloud_code_generator.cmd.refresh_examples```
 
 ## How to refresh the amazon.cloud collection
 
-Fork the original [``amazon.cloud collection``](https://github.com/ansible-collections/amazon.cloud) repository to your account on GitHub. Clone the repository from your fork:
+Fork the original [``amazon.cloud collection``](https://github.com/ansible-collections/amazon.cloud) repository to your account on GitHub. Ensure to have the path `~/.ansible/collections/ansible_collections/amazon/cloud` and clone the repository from your fork:
 ```
 mkdir -p ~/.ansible/collections/ansible_collections/amazon/cloud
 git clone https://github.com/YOUR_USER/amazon.cloud ~/.ansible/collections/ansible_collections/amazon/cloud
 ```
 
-Refresh the content of the modules moving in the repository path:
+Refresh the content of the modules moving in the repository path using `tox -e refresh_modules`.
+
+This command will refresh the content of the ``amazon.cloud collection`` plugins including EXAMPLE block and will format the plugins using the black formatter.
 ```
 cd ~/.ansible/collections/ansible_collections/amazon/cloud
 tox -e refresh_modules
 ```
-Format the Python code of the modules using the black formatter:
+
+You can also specify the next release version by passing the `--next-version` parameter as follows:
+```
+tox -e refresh_modules --next-version 0.2.0
+```
+
+Individual opperations are also allowed. 
+
+For example, you can only format the Python code of the plugins using the black formatter by running:
 
 ```tox -e black```
 
-Refresh the modules documentation localed in `~/.ansible/collections/ansible_collections/amazon/cloud/docs`.
+You can also refresh the plugins documentation localed in `~/.ansible/collections/ansible_collections/amazon/cloud/docs` just by running.
```

Comment:
```suggestion
You can also refresh the plugins documentation located in `~/.ansible/collections/ansible_collections/amazon/cloud/docs` just by running.
```

Reply:
Rephrased like this: ``Refresh the content of the modules by moving in the repository path `~/.ansible/collections/ansible_collections/amazon/cloud` and running `tox -e refresh_modules`.
``